.circleci/run.sh

```diff
@@ -145,8 +160,16 @@ codecov()
 
 case $1 in
     install-deps) install_deps ;;
-    setup-repos) setup_repos ;;
-    coverage) coverage ;;
-    check-clean-git) check_clean_git;;
-    codecov) codecov ;;
+    setup-repos) echo "removed" ;;
+    coverage) echo "removed" ;;
+    check-clean-git) echo "removed" ;;
+    codecov)
+        echo "removed - use 'all'"
+        ;&
```

Comment:
fall-through is required, s.t. existing PRs don't need to be rebased, but will call `all` automatically.

Reply:
Probably a good idea to add that as a comment to the script.

Reply:
Yes! Done.